### PR TITLE
campaigns: Return scheduled sync time in API

### PR DESCRIPTION
**Pull request by ryanslade**
Closes: https://github.com/sourcegraph/sourcegraph/issues/9108

### CODE REVIEW

**Comment by ryanslade** [comment]
> Do you wish me adding it to the UI or is this a more internally used feature?

Happy for you to add it. I actually tried but I kept saying "invalid time value". They look fine to me when querying the API so I'm not sure what the issue was

**Comment by ryanslade** [comment]
I'll merge this now, no rush for the UI changes

**Comment by keegancsmith** [comment]
meta: can you include the motivation for adding this in the description. Linking to the issue is alright, but its nice to have it self contained (especially when browsing commits locally).

File: cmd/frontend/graphqlbackend/schema.graphql [hunk 1]
```diff
@@ -755,6 +755,9 @@ type ExternalChangeset implements Node {
     # The date and time when the changeset was updated.
     updatedAt: DateTime!
 
+    # The date and time when the next changeset sync is scheduled
+    nextSync: DateTime!
```
Comment (unknwon) [inline cmd/frontend/graphqlbackend/schema.graphql:759]: Any reason we don't use `nextSyncAt`? Seems a naming convention for DateTime/time.Time type.

Comment (ryanslade) [reply cmd/frontend/graphqlbackend/schema.graphql:759]: Good point, I'll send a PR as nobody but us are depending on this field

**Pull request merged**